carbonmark-api/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@klimadao/carbonmark-api",
-  "version": "4.1.3",
+  "version": "4.2.0",
```

Comment:
oh, due to that breaking change with the country field, this needs to be a Major bump actually.

Reply:
Done in 2b927577f